client/client-core/src/main/java/com/cognifide/aet/common/ProcessingErrorStatusProcessor.java

```diff
@@ -18,23 +18,19 @@
 package com.cognifide.aet.common;
 
 import com.cognifide.aet.communication.api.exceptions.AETException;
-import com.cognifide.aet.communication.api.messages.ProcessingErrorMessage;
+import com.cognifide.aet.communication.api.suiteexecution.SuiteStatusResult;
 import com.jcabi.log.Logger;
 
-public class ProcessingErrorMessageProcessor implements MessageProcessor {
+public class ProcessingErrorStatusProcessor implements StatusProcessor {
 
-  private final ProcessingErrorMessage data;
+  private final SuiteStatusResult suiteStatusResult;
 
-  public ProcessingErrorMessageProcessor(ProcessingErrorMessage data) {
-    this.data = data;
+  public ProcessingErrorStatusProcessor(SuiteStatusResult suiteStatusResult) {
+    this.suiteStatusResult = suiteStatusResult;
   }
 
   @Override
   public void process() throws AETException {
-    if (data != null && data.getProcessingError() != null) {
-      Logger.warn(this, data.getProcessingError().getDescription());
-    } else {
-      throw new AETException("ProcessingErrorMessageProcessor has received empty message!");
-    }
+    Logger.warn(this, suiteStatusResult.getMessage());
```

Comment:
`Logger` variable name inconsistent with other classes in the PR where it's called `LOGGER`